IntelliTect.Analyzer/IntelliTect.Analyzer.Test/Verifiers/DiagnosticVerifier.cs

```diff
@@ -1,11 +1,22 @@
 using System;
-using Microsoft.CodeAnalysis;
+using System.Collections.Generic;
+
+/* Unmerged change from project 'IntelliTect.Analyzer.Tests (net6.0)'
```

Comment:
Do we need this commented code for anything?

Reply:
Code Cleanup remnants... thanks